lib/counter_culture/collection.rb

```diff
@@ -0,0 +1,45 @@
+module CounterCulture
+  class Collection
+    include Enumerable
+
+    def initialize(arr=Array.new)
+      @arr = arr
+    end
+
+    def fix_counts(options={})
+      options[:exclude] = [options[:exclude]] if options[:exclude] && !options[:exclude].is_a?(Enumerable)
+      options[:exclude] = options[:exclude].try(:map) {|x| x.is_a?(Enumerable) ? x : [x] }
+      options[:only] = [options[:only]] if options[:only] && !options[:only].is_a?(Enumerable)
+      options[:only] = options[:only].try(:map) {|x| x.is_a?(Enumerable) ? x : [x] }
+
+      flat_map do |counter|
+        next if options[:exclude] && options[:exclude].include?(counter.relation)
+        next if options[:only] && !options[:only].include?(counter.relation)
+
+        reconciler = CounterCulture::Reconciler.new(counter, options.slice(:skip_unsupported))
+        reconciler.reconcile!
+        reconciler.changes
+      end.compact
+    end
+
+    def each(&block)
```

Comment:
instead of using enumerable? Certainly could but I am also exposing the enumerable methods publicly for consistency of a collection concept.